enhancements/worker-latency-profile/worker-latency-profile.md

```diff
@@ -0,0 +1,450 @@
+| Kube API Server         | --default-unreachable-toleration-seconds |    300s     |
+
+
+#### Medium Update And Average Reaction
+
+While the default configuration works well in most cases, sometimes the worker nodes may find themselves in a network with slightly higher than usual latency.
+
+In this scenario, we will reduce the frequency of `Kubelet` updates to every 20 seconds (`--node-status-update-frequency`) and change the node monitor grace period of the `Kube Controller Manager` to 2 minutes (`--node-monitor-grace-period`). `--default-not-ready-toleration-seconds` and `--default-unreachable-toleration-seconds` of the Kube API Server will be set to 60 seconds.
+
+`Kube Controller Manager` will wait for 2 minutes to consider unhealthy status for the node and since `--default-not-ready-toleration-seconds` and `--default-unreachable-toleration-seconds` of the Kube API Server are set to 60 seconds the total time will be 3 minutes before eviction process starts.
+
+
+| Component               | Flag Name                                | Flag Value  |
+| -----------             | -----------                              | ----------- |
+| Kubelet                 | --node-status-update-frequency           |    20s      |
+| Kube Controller Manager | --node-monitor-grace-period              |    2m       |
+| Kube API Server         | --default-not-ready-toleration-seconds   |    60s      |
+| Kube API Server         | --default-unreachable-toleration-seconds |    60s      |
+
+#### Low Update and Slow reaction
+
+Worker nodes may find themselves in a network with extremely high latency and/or bad reliability.
+In this scenario, we will reduce the frequency of `Kubelet` updates further to every 1 minute (`--node-status-update-frequency`) and change the node monitor grade period of the `Kube Controller Manager` to every 5 minutes (`--node-monitor-grace-period`). `--default-not-ready-toleration-seconds` and `--default-unreachable-toleration-seconds` of the Kube API Server are set to 60 seconds.
+
+`Kube Controller Manager` will wait for 5 minutes to consider unhealthy status for the node and since `--default-not-ready-toleration-seconds` and `--default-unreachable-toleration-seconds` of the `Kube API Server` are set to 60 seconds the total time will be 6 minutes before eviction process starts.
+
+
+| Component               | Flag Name                                | Flag Value  |
+| -----------             | -----------                              | ----------- |
+| Kubelet                 | --node-status-update-frequency           |    1m       |
+| Kube Controller Manager | --node-monitor-grace-period              |    5m       |
+| Kube API Server         | --default-not-ready-toleration-seconds   |    60s      |
+| Kube API Server         | --default-unreachable-toleration-seconds |    60s      |
+
+
+#### Failure Modes
+1. In case of failure `WorkerLatencyProfileStatus` should point towards failed component(s)
```

Comment:
See my other comment about reporting status, in the API section.

Reply:
Thanks, I replied there https://github.com/openshift/enhancements/pull/977#discussion_r782692490